projects/librdkafka.cmake

```diff
@@ -1,13 +1,13 @@
 ########################################
-# qt5
+# librdkafka
 ########################################
 xpProOption(librdkafka)
-set(KAFKA_VER 0.9.0)
+set(KAFKA_VER master)
 set(KAFKA_REPO https://github.com/edenhill/librdkafka)
```

Comment:
Looks like I was being all clever and already forked librdkafka to distributepro: https://github.com/distributePro/librdkafka  We should probably update that and then build from that going forward.